src/ports/squids/component.ts

```diff
@@ -0,0 +1,184 @@
+import { ISquidComponent, Squid, SquidMetric } from "./types";
+import {
+  ECSClient,
+  ListServicesCommand,
+  ListTasksCommand,
+  DescribeTasksCommand,
+  ListServicesRequest,
+  UpdateServiceCommand,
+  DescribeServicesCommand,
+} from "@aws-sdk/client-ecs";
+import {
+  IConfigComponent,
+  IFetchComponent,
+} from "@well-known-components/interfaces";
+import { IPgComponent } from "@well-known-components/pg-component";
+import { getPromoteQuery } from "./queries";
+import { getMetricValue, getSquidsNetworksMapping } from "./utilts";
+import { Network } from "@dcl/schemas";
+
+const AWS_REGION = "us-east-1";
+
+export async function createSubsquidComponent({
+  fetch,
+  dappsDatabase,
+  config,
+}: {
+  fetch: IFetchComponent;
+  dappsDatabase: IPgComponent;
+  config: IConfigComponent;
+}): Promise<ISquidComponent> {
+  const cluster: string =
+    (await config.getString("CLUSTER_NAME")) || "dev-main";
```

Comment:
why dev-main here? could we fail if CLUSTER_NAME is empty?

Reply:
changed to `requireString` to ensure it has value or it explodes.